### PR TITLE
pin httpx update

### PR DESCRIPTION
**Pull request by BinamB**
### Dependency updates
- Update httpx to ^0.14.0

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ cdiserrors = "~=1.0"
 xmltodict = "~=0.9"
 
 authlib = "~=0.11"
-httpx = "^0.12.1"
```
Comment (paulineribeyre) [inline pyproject.toml:15]: can't poetry resolve `"^0.14.0" + "^0.12.1"` on its own?

Comment (BinamB) [reply pyproject.toml:15]: oh yeah it should. `^0.12.0` is basically `>=0.12.0` right?

Comment (paulineribeyre) [reply pyproject.toml:15]: yeah, you shouldn't need this PR 🤔

Comment (BinamB) [reply pyproject.toml:15]: deleting this PR :+1:

**Pull request closed without merging**